tools/docker-compose/ansible/roles/sources/tasks/main.yml

```diff
@@ -8,6 +8,7 @@
   loop:
     - secrets
     - receptor
+    - work_sign_keys
```

Comment:
we may just want to put this stuff under the receptor, or in a subfolder within receptor. that way people know these keys are a receptor thing

Reply:
I have made the required change.

Reply:
i think you still need to remove this - work_sign_keys, the folder is still be created

Reply:
I have made the changes.